api/client.go

```diff
@@ -36,6 +38,10 @@ func Client(config jira.Config) *jira.Client {
 			config.APIToken = netrcConfig.Password
 		}
 	}
+	if config.APIToken == "" {
+		secret, _ := keyring.Get("jira-cli", "token")
```

Comment:
Instead of `token` string, it might be better to use login email/username as a key like we do it with `.netrc`. This way you can configure different token for different user.

```suggestion
		secret, _ := keyring.Get("jira-cli", config.Login)
```